CONTRIBUTING.md

```
@@ -565,10 +565,59 @@ Appwrite uses [XDebug](https://github.com/xdebug/xdebug) debugger, which can be
+        }
+        ...
+    ]
+}
+```
+
+### PHPStorm Setup
+
+In settings, go to **Languages & Frameworks** > **PHP** > **Debug**, under **Xdebug** set the debug port to **9003** and enable the **can accept external connections** checkbox.
+
+5. Start the debugger listening.
+6. Launch the Appwrite vai `docker-compoise up `.
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)